### ISSUE TITLE
POST next.js route await req.text() returns "[object EventEmitter]" on docker oven/bun

### ISSUE THREAD

**Issue by snomiao** [oven-sh/bun#7806]
### What version of Bun is running?

1.0.19

### What platform is your computer?

Linux 5.15.0-86-generic x86_64 unknown, running on docker oven/bun

## What happened

Create a simple next.js app route, and dev with docker oven/bun

```ts
// app/route.ts
export const POST = async (req: Request) =>
  new Response("got-post-data: " + (await req.text()));

// the await req.text()
// returns correct post data on node.js ✅ (expected)
// returns "[object EventEmitter]" ❌ while run with "bun dev" in oven/bun
```

Try post any data into this route, like `curl --data expected-data http://localhost:3000`

It should got expected stdout: `got-post-data: expected-data`

But actual got  stdout: `got-post-data: [object EventEmitter]`

## Reproduce Repo

1. Clone this reproduce repo: https://github.com/snomiao/reproduce-bun-nextjs-event-emitters

2. Reproduce
Run `docker compose up`

expected stdout: got-post-data: expected-data
but...
actual   stdout: got-post-data: [object EventEmitter]

3. Confirm Problem

goto `docker-compose.yml`, replace the image `oven/bun` with `node`, you will got expected result

```yaml
services:
  reproduce-bun-nextjs-event-emitters:
    # reproduce image and commands
    # image: oven/bun
    # command: bun dev -H 0.0.0.0 -p 80

    # replace with this, you will got expected result
    image: node
    command: npx next -H 0.0.0.0 -p 80
```

## Reference

reproduce repo: https://github.com/snomiao/reproduce-bun-nextjs-event-emitters

### What is the expected behavior?

the await req.text()
returns correct post data on node.js ✅ (expected)

### What do you see instead?

the await req.text()
returns "[object EventEmitter]" ❌ while run with "bun dev" in oven/bun

### Additional information

reproduce repo: https://github.com/snomiao/reproduce-bun-nextjs-event-emitters

**Comment by LordZardeck** [comment]
This is happening to us as well, causing all our payloads to fail

**Comment by snomiao** [comment]
Problem solved, confirmed with bun v1.1.17
